apps/dashboard/app/helpers/files_helper.rb

```diff
@@ -1,5 +1,7 @@
 # Helper for /files pages.
 module FilesHelper
+  include ApplicationHelper
```

Comment:
Do we need this? Is it for `html_safe`? If so, I'm not sure if we need `html_safe` on a `link_to` method if `files_button` is being called in a partial.